Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -407,6 +407,21 @@ namespace AZ
 
             SceneNotificationBus::Event(GetId(), &SceneNotification::OnBeginPrepareRender);
 
+            // Get active pipelines which need to be rendered and notify them of an impending frame.
+            AZStd::vector<RenderPipelinePtr> activePipelines;
+            {
+                // FIXME: this crashes the CPU profiler, need to figure out why
```

Comment:
This needs a Jira or GHI number so it doesnt get lost.

Reply:
Found and fixed it (fun with instrumenting part of the render pipeline from within IMGUI...)